### ISSUE TITLE
Holodeck Boxing Damage doesn't show in Autopsy Scanner

### ISSUE THREAD

**Issue by niennab** [Aurorastation/Aurora.3#7448]
During a round where two characters boxed in the Holodeck, one character didn't pull punches and got overly aggressive, prompting the other character to do the same, resulting in the death of one person. IC issues aside, when I brought in the body and conducted an Autopsy as my Forensic Technician, the Autopsy Scanner provided only the time of death and no information on the blunt force trauma.

I thought I might not have opened the surgery site however in attempting to 'open it', I stabbed the body, which effectively means the site was open.

The screenshot shows me scanning them at the top and stabbing them at the bottom.

![autopsybug](https://user-images.githubusercontent.com/56227386/69066922-8bb39c00-0a65-11ea-889f-8d1094e8ab51.png)

**Comment by alsoandanswer** [comment]
If it didn't show up as, "you cut open them with a scalpel" in blue text, it basically means that you haven't technically started the "surgery" procedure and the code for the autopsy scanner cannot continue normally. 

If you only hit him with the scalpel (perhaps you were on harm intent), then it's the likely cause.

**Comment by niennab** [comment]
I don't believe I had harm intent on. I also found that today I was unable to conduct an autopsy on another patient. 
![Screenshot 2019-11-19 21 53 09](https://user-images.githubusercontent.com/56227386/69148303-549dc300-0b17-11ea-94db-e83f9655a31d.png)

However hitting them with the scalpel gave me information of that strike.
